bin/tronctl

```diff
@@ -227,9 +228,14 @@ def main():
     cmd_utils.setup_logging(args)
     cmd_utils.load_config(args)
     cmd = COMMANDS[args.command]
-    for ret in cmd(args):
-        if not ret:
-            sys.exit(ExitCode.fail)
+    try:
+        for ret in cmd(args):
+            if not ret:
+                sys.exit(ExitCode.fail)
+    except RequestError as err:
+        print(f"Error connecting to tron: {err}", file=sys.stderr)
```

Comment:
What if you just print the server it's trying to connect to, from the args?

Reply:
Can do